### PR TITLE
:memo: DOC: Improve Installation Instructions

### PR DESCRIPTION
**Pull request by shaneahmed**
- Improve installation instructions

Documentation link: https://tia-toolbox.readthedocs.io/en/doc-improve-installation-instructions/installation.html

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -36,50 +32,31 @@ using
...
+^^^^^^^^
 
 .. code-block:: console
 
     $ port install openjpeg openslide
 
 
-Stable release
---------------
+Installing Stable release (Recommended)
```
Comment (John-P) [inline docs/installation.rst:58]:
```suggestion
Installing Stable Release (Recommended)
```

File: docs/installation.rst [hunk 2]
```diff
@@ -36,50 +32,31 @@ using
...
+=======================================
 
 Please note that TIAToolbox is tested for python version 3.8, 3.9 and 3.10.
 To install TIA Toolbox, run this command in your terminal after you have installed the prerequisite software:
```
Comment (John-P) [inline docs/installation.rst:62]:
```suggestion
To install TIAToolbox, run this command in your terminal after you have installed the prerequisite software:
```

File: docs/installation.rst [hunk 3]
```diff
@@ -96,13 +73,33 @@ To upgrade an existing version of tiatoolbox to the latest stable release, run t
...
 .. _Python installation guide: http://docs.python-guide.org/en/latest/starting/installation/
 
 
+Alternative Method(s)
+=====================
+
+Using Anaconda
+--------------
+
+After `installing Anaconda <https://docs.anaconda.com/anaconda/install/index.html>`_ (or miniconda), you can install TIA toolbox using the following command:
```
Comment (John-P) [inline docs/installation.rst:95]: Add link to miniconda?

Comment (John-P) [inline docs/installation.rst:95]:
```suggestion
After `installing Anaconda <https://docs.anaconda.com/anaconda/install/index.html>`_ (or miniconda / mamba), you can install TIA toolbox using the following command:
```

Comment (John-P) [reply docs/installation.rst:95]: Mamba: https://mamba.readthedocs.io/en/latest/user_guide/mamba.html

Comment (shaneahmed) [inline docs/installation.rst:95]:
```suggestion
After installing `Anaconda <https://docs.anaconda.com/anaconda/install/index.html>`_ (or `miniconda <https://docs.conda.io/en/latest/miniconda.html>`), you can install TIA toolbox using the following command:
```

File: docs/installation.rst [hunk 4]
```diff
@@ -96,13 +73,33 @@ To upgrade an existing version of tiatoolbox to the latest stable release, run t
...
 .. _Python installation guide: http://docs.python-guide.org/en/latest/starting/installation/
 
 
+Alternative Method(s)
+=====================
+
+Using Anaconda
+--------------
+
+After installing `Anaconda <https://docs.anaconda.com/anaconda/install/index.html>`_ (or `miniconda <https://docs.conda.io/en/latest/miniconda.html>`), you can install TIA toolbox using the following command:
```
Comment (shaneahmed) [inline docs/installation.rst:95]:
```suggestion
After installing `Anaconda <https://docs.anaconda.com/anaconda/install/index.html>`_ (or `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ ), you can install TIAToolbox using the following command:
```

File: docs/installation.rst (hunk 3 above)
Comment (shaneahmed) [reply docs/installation.rst]: Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/567/commits/6f77826d6fc71f40b554e4c36f7e1c97db97f698

[the remark this answers is not in the document]
Comment (shaneahmed) [reply docs/installation.rst]: Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/567/commits/6f77826d6fc71f40b554e4c36f7e1c97db97f698

File: docs/installation.rst [hunk 5]
```diff
@@ -96,13 +73,33 @@ To upgrade an existing version of tiatoolbox to the latest stable release, run t
...
 .. _Python installation guide: http://docs.python-guide.org/en/latest/starting/installation/
 
 
+Alternative Method(s)
+=====================
+
+Using Anaconda
+--------------
+
+After installing `Anaconda <https://docs.anaconda.com/anaconda/install/index.html>`_ (or `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ ), you can install TIAToolbox using the following command:
```
Comment (shaneahmed) [inline docs/installation.rst:95]:
```suggestion
After installing `Anaconda <https://docs.anaconda.com/anaconda/install/index.html>`_ (or `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ / `mamba <https://mamba.readthedocs.io/en/latest/user_guide/mamba.html>`_ ), you can install TIAToolbox using the following command:
```

File: docs/installation.rst [hunk 6]
```diff
@@ -96,13 +73,33 @@ To upgrade an existing version of tiatoolbox to the latest stable release, run t
...
 .. _Python installation guide: http://docs.python-guide.org/en/latest/starting/installation/
 
 
+Alternative Method(s)
+=====================
+
+Using Anaconda
+--------------
+
+After installing `Anaconda <https://docs.anaconda.com/anaconda/install/index.html>`_ (or `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ / `mamba <https://mamba.readthedocs.io/en/latest/user_guide/mamba.html>`_ ), you can install TIAToolbox using the following command:
```
Comment (shaneahmed) [inline docs/installation.rst:95]:
```suggestion
After installing `Anaconda <https://docs.anaconda.com/anaconda/install/index.html>`_ (or `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ / `mamba <https://mamba.readthedocs.io/en/latest/user_guide/mamba.html#mamba-vs-conda-clis>`_ ), you can install TIAToolbox using the following command:
```

File: docs/installation.rst (hunk 3 above)
Comment (shaneahmed) [reply docs/installation.rst]: Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/567/commits/187f81540e45c1faae08649dcc710a4090812eda

File: docs/installation.rst [hunk 7]
```diff
@@ -96,13 +73,33 @@ To upgrade an existing version of tiatoolbox to the latest stable release, run t
...
 .. _Python installation guide: http://docs.python-guide.org/en/latest/starting/installation/
 
 
+Alternative Method(s)
+=====================
+
+Using Anaconda
+--------------
+
+After installing `Anaconda <https://docs.anaconda.com/anaconda/install/index.html>`_ (or `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ / `mamba <https://mamba.readthedocs.io/en/latest/user_guide/mamba.html#mamba-vs-conda-clis>`_ ), you can install TIAToolbox using the following command:
```
Comment (John-P) [inline docs/installation.rst:95]:
```suggestion
After installing `Anaconda <https://docs.anaconda.com/anaconda/install/index.html>`_ (or `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ / `mamba <https://mamba.readthedocs.io/en/latest/user_guide/mamba.html#mamba-vs-conda-clis>`_), you can install TIAToolbox using the following command:
```

Just removing the space at the end of the brackets.

File: docs/installation.rst [hunk 8]
```diff
@@ -131,22 +128,22 @@ Once you have a copy of the source, you can install it with:
...
 To run TIA toolbox in an isolated environment, use our `Docker image <https://github.com/tissueimageanalytics/tiatoolbox-docker/pkgs/container/tiatoolbox>`_ . We host different Dockerfiles in our github repository `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>`_. Please report any issues related to the docker image in the repository `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>`_.
 
 After `installing Docker <https://docs.docker.com/get-docker/>`_ (or Docker Desktop), you can use our TIA toolbox image in 3 different ways.
 
-Use the pre-built docker image
+Use the Pre-Built Docker Image
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-1. Pull the image from the Github Container Registry
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+1. Pull the Image From Github Container Registry
```
Comment (shaneahmed) [inline docs/installation.rst:139]: TIAToolbox provides pre-built docker containers which can be downloaded using the instructions below:

Comment (shaneahmed) [inline docs/installation.rst:139]:
```suggestion
TIAToolbox provides pre-built docker containers which can be downloaded using the instructions below:
1. Pull the Image From Github Container Registry
```

File: docs/installation.rst [hunk 9]
```diff
@@ -131,22 +128,23 @@ Once you have a copy of the source, you can install it with:
...
 
 After `installing Docker <https://docs.docker.com/get-docker/>`_ (or Docker Desktop), you can use our TIA toolbox image in 3 different ways.
 
-Use the pre-built docker image
+Use the Pre-Built Docker Image
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-1. Pull the image from the Github Container Registry
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+TIAToolbox provides pre-built docker containers which can be downloaded using the instructions below:
+1. Pull the Image From Github Container Registry
```
Comment (shaneahmed) [inline docs/installation.rst:140]:
```suggestion

1. Pull the Image From Github Container Registry
```

File: docs/installation.rst [hunk 10]
```diff
@@ -140,13 +137,13 @@ After `installing Docker <https://docs.docker.com/get-docker/>`_ (or Docker Desk
 Use the pre-built docker image
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 1. Pull the image from the Github Container Registry
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+""""""""""""""""""""""""""""""""""""""""""""""""""""
 .. code-block:: console
 
     $ docker pull ghcr.io/tissueimageanalytics/tiatoolbox:latest
 
 2. Use the pre-built Docker image as a base image in a Dockerfile
```
[the remark this answers is not in the document]
Comment (shaneahmed) [reply docs/installation.rst]: Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/567/commits/1ba477930f4a4bb0b30d55f24a08111a756fe4cb

File: docs/installation.rst [hunk 11]
```diff
@@ -140,13 +137,13 @@ After `installing Docker <https://docs.docker.com/get-docker/>`_ (or Docker Desk
 Use the pre-built docker image
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 1. Pull the image from the Github Container Registry
```
[the remark this answers is not in the document]
Comment (shaneahmed) [reply docs/installation.rst]: Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/567/commits/0489b845109b8e3702b11a1b27eac82bf6dce012

File: docs/installation.rst [hunk 12]
```diff
@@ -140,13 +137,13 @@ After `installing Docker <https://docs.docker.com/get-docker/>`_ (or Docker Desk
 Use the pre-built docker image
```
[the remark this answers is not in the document]
Comment (shaneahmed) [reply docs/installation.rst]: Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/567/commits/0489b845109b8e3702b11a1b27eac82bf6dce012

File: docs/installation.rst [hunk 13]
```diff
@@ -36,73 +32,83 @@ using
...
 
-Stable release
---------------
+Installing Stable Release (Recommended)
+=======================================
 
 Please note that TIAToolbox is tested for python version 3.8, 3.9 and 3.10.
-To install TIA Toolbox, run this command in your terminal after you have installed the prerequisite software:
+
+New
```
Comment (mostafajahanifar) [inline docs/installation.rst:63]: I don't think having a subheader of "New" is required here. Just have subheaders for upgrades and without dependencies.

**Review by mostafajahanifar — APPROVED** [review]
Thank you very much, Shan. No installation is more clear and would not confuse (or stale users for Conda solving env) anymore. 

I just suggest removing the "New" title because it seems redundant now.

File: docs/installation.rst (hunk 13 above)
Comment (shaneahmed) [reply docs/installation.rst:63]: ok let me change it to Recommended and remove recommended from the title.

File: docs/installation.rst [hunk 14]
```diff
@@ -36,73 +32,83 @@ using
...
+^^^^^^^^
 
 .. code-block:: console
 
     $ port install openjpeg openslide
 
 
-Stable release
---------------
+Installing Stable Release (Recommended)
```
Comment (shaneahmed) [inline docs/installation.rst:58]:
```suggestion
Installing Stable Release
```

File: docs/installation.rst (hunk 13 above)
Comment (shaneahmed) [inline docs/installation.rst:63]:
```suggestion
Recommended
```

Comment (shaneahmed) [reply docs/installation.rst]: Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/567/commits/d221593ae51a15d2743d3b60a0bcb2ceee7344fb

File: docs/installation.rst [hunk 15]
```diff
@@ -36,73 +32,83 @@ using
...
 
 .. code-block:: console
 
     $ port install openjpeg openslide
 
 
-Stable release
---------------
+Installing Stable Release
+=======================================
```
Comment (shaneahmed) [inline docs/installation.rst:59]:
```suggestion
=========================
```

File: docs/installation.rst [hunk 16]
```diff
@@ -36,73 +32,83 @@ using
...
-Stable release
---------------
+Installing Stable Release
+=========================
 
 Please note that TIAToolbox is tested for python version 3.8, 3.9 and 3.10.
-To install TIA Toolbox, run this command in your terminal after you have installed the prerequisite software:
+
+Recommended
+---
```
Comment (shaneahmed) [inline docs/installation.rst:64]:
```suggestion
-----------
```

File: docs/installation.rst [hunk 17]
```diff
@@ -4,30 +4,26 @@ Installation
...
+    $ apt-get -y install libopenjp2-7-dev libopenjp2-tools openslide-tools
 
-If you cannot use Anaconda or are having trouble with it, you can try an alternative install method. We will install prerequisite binary packages and then use pip (the Python package manager) to install python dependencies.
+The same command is used when working on the Colab or Kaggle platforms.
+When working on Google Colab, we remove the packages ``datascience`` and ``albumentations`` because they conflict
+and produce an error message.
 
-Windows
-^^^^^^^
+Windows (10 and 11)
```
Comment (shaneahmed) [inline docs/installation.rst:23]:
```suggestion
Windows (10+)
```

**Pull request merged**